### PR TITLE
Delmar update

### PR DESCRIPTION
**Pull request by jhewitt12**

### CODE REVIEW

**Comment by jhewitt12** [comment]
The scss compiles correctly now and the hero unit is back to normal. Could you put this version on the dev server more me to test? Thanks!

**Comment by elliottwilliams** [comment]
Yep. Just so you know, adding commits to a branch adds them to the open pull request, so you'll confuse people if you close a PR and immediately open a new one :)

**Comment by elliottwilliams** [comment]
https://github.com/matchboxcowork/matchboxstudio.org/pull/31/commits/81b10de6db105c0f00e1f4ae6ef888d519cb66c0 is live at http://104.131.179.41/ as of `Fri Jan 13 16:05:34 EST 2017`.

**Comment by elliottwilliams** [comment]
> Things are looking good overall except it can't find newsCtl.php for some reason. Any ideas? It says this file is not found "http://104.131.179.41/newsCtl.php". But its in the top level of the build folder. Do I need to reference it to somewhere else? Getting close!

It should be getting picked up by guard when building, since it's a `.haml` file in the root directory. I'm taking a look at this, will keep you updated.

**Comment by elliottwilliams** [comment]
The issue had to do with Guard (the runner that compiles all dependencies) trying to compile stuff from the `vendor` directory, which is created on production when I ran `bundle install --deployment`. Guard would fail to compile a haml file from within `vendor`, which silently aborted the haml stage of the build.

I pushed a fix to `master` to take care of it. To add it to your branch, do a rebase (`git rebase master`) and a force push (`git push origin delmar_update --force`). Force push is necessary since the rebase modifies commit history.

**Comment by jhewitt12** [comment]
For some reason its saying the news table doesn't exist in the database when on the home page and news-admin.html page. I never had this issue before. When the page loads, there's an ajax call to newsCtl.php that retrieves the news articles. Did something you changed effect it? I checked my local dev_db.sqlite file and the table is in there. Any ideas? Sorry to be so difficult. I had it all working locally and didn't expect these problems when getting it on the server.

**Comment by elliottwilliams** [comment]
No, I was just noticing the same thing. It looks like you changed the dev_db path in https://github.com/matchboxcowork/matchboxstudio.org/pull/31/commits/12f3d3f6ea0b0521c85269ec6336665cd56ab0f9, was that intentional? Changing it back fixed things for me.

**Comment by jhewitt12** [comment]
I remember changing that a long time ago to get it to work. I changed it back and it worked for me too. Okay fingers crossed thats its good now. Could you update the dev server with the latest commit? Thanks!

**Comment by elliottwilliams** [comment]
Sure. https://github.com/matchboxcowork/matchboxstudio.org/pull/31/commits/fdd89c174b9ab6dbdd87b363bd95e1e53508fe45 is live on http://104.131.179.41/ as of `Thu Jan 19 16:05:14 EST 2017`.

**Comment by elliottwilliams** [comment]
Since it seems like you've got a build working, I'm going to leave some suggestions in your code while we wait to hear back from Jason. Once you have an okay from him, let me know and I'll go about merging and deploying.

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ config/prod.php
 dev_db.sqlite3
 build/
 .sass-cache
+assets/vid/IIB 10-06-16 Segment 5A.mp4
```
Comment (elliottwilliams) [inline .gitignore]: What's going on here? If this video doesn't need to exist in your source tree, it shouldn't be in the repo. Did you at one point try to upload the video before deciding to use vimeo?

**Comment by jhewitt12** [comment]
Feel free to leave some suggestions. I'm getting an error with newsCtl.php on http://104.131.179.41/. When I got to  http://104.131.179.41/newsCtl.php I see this error `Fatal error: Uncaught exception 'PDOException' with message 'could not find driver' in /var/www/matchboxstudio.org/build/newsCtl.php:8 Stack trace: #0 /var/www/matchboxstudio.org/build/newsCtl.php(8): PDO->__construct('sqlite:/var/www...', '', '') #1 {main} thrown in /var/www/matchboxstudio.org/build/newsCtl.php on line 8`. Do you get the same thing? How come it works locally but not live?

File: create_news.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+CREATE TABLE news (
+    id          INTEGER        PRIMARY KEY AUTOINCREMENT,
+    [order]     VARCHAR (1000),
+    title       VARCHAR (255),
+    href        VARCHAR (500),
+    description TEXT (1000),
+    active      BOOLEAN,
+    deleted     BOOLEAN
+);
```
Comment (elliottwilliams) [inline create_news.sql]: Let's keep things organized to accomodate for future changes to the database. Could you:

- rename this file to include a `YYYY-MM-dd` date? i.e. `2017-01-19_create_news.sql`
- move this file to a new `migrations` directory

That way, as new DB changes come in it'll be easier for us to keep track of what has and hasn't been applied.

File: news-admin.haml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    %section.footer
+      .container
+        .sixteen.columns
+          MatchBOX Coworking Studio © All Rights Reserved
+
+    / End Content
+
+    / JS
+    =include 'partials/body_scripts.haml'
+
```
Comment (elliottwilliams) [inline news-admin.haml]: Could you move this page and `newsCtl.php.haml` into the `admin` directory along with the other management-related pages?

File: newsCtl.php.haml [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    $sql = "UPDATE news SET 'order'='".$order."' WHERE id = '".$id."'";
+
+    if($dbh->query($sql) === TRUE)
+    {  
+      //echo "Success";
+    }
+    getArticles($dbh);
+
+  }
+
```
Comment (elliottwilliams) [inline newsCtl.php.haml]: See above comment.

File: partials/pricing_and_application.haml [hunk 5]
```diff
@@ -91,7 +91,7 @@
 
 .container
   %h2.sixteen.columns.message-block
-    %a.button.mbxred{:href => "#app"}
+    %a.button.mbxred{:href => "/about.html"}
```
Comment (elliottwilliams) [inline partials/pricing_and_application.haml]: Did you mean to link to `/apply.html`?

File: sass/_imported.scss [hunk 6]
```diff
@@ -251,5 +432,11 @@ img.fullimage {
...
     }
-}
 
+    .about-paragraphs .eight:first-child {
+        padding-right:0;
+    }
+    .about-paragraphs .eight:nth-child(2) {
+        padding-left:0;
+    }
+}
```
Comment (elliottwilliams) [inline sass/_imported.scss]: Sorry I hadn't been clearer about this, but could you move all about-page related scss into its own file?`_imported.scss` contains holdover from the original template, and as you can tell it's pretty unorganized.

Instead, create something like `_about.scss` in the `sass` directory, then add another import statement at the bottom of `mbx.scss`. The underscore in the filename tells Sass not to compile a standalone `about.css`, but rather use it as an import for other files. So importing a stylesheet into `mbx.scss` causes it to end up in the generated `mbx.css`.

**Comment by elliottwilliams** [comment]
Looking into it — probably something not installed / configured wrong on the test server.

**Comment by elliottwilliams** [comment]
Yeah, it was a missing php module. Should be working now.

**Review by elliottwilliams — CHANGES_REQUESTED** [review]
Some organizational nitpicks, but overall this looks good. Let me know if I can better clarify any of my comments.

File: .gitignore (hunk 1 above)
Comment (jhewitt12) [reply .gitignore]: Originally it was in the repo but then I realized it was too large to upload to Github. So I deleted it from the repo and had Mikel upload it to Vimeo. I tried to push to Github again and it still gave me errors. Apparently it was still in the history of commits and I had all kinds of trouble trying to remove it from the history. After surfing through forums I finally found the right command and removed it. I'll take that out of the .gitignore.

**Comment by jhewitt12** [comment]
I put the news admin pages into the admin folder. I have the username and password to access it but I still get "Server error! Error 500" when trying to load the page so I'm unable to test if it works in its new directory. I'm worried file path relativity may have broken.

**Review by elliottwilliams — APPROVED** [review]

**Comment by elliottwilliams** [comment]
See the commit I added to use absolute paths in partials and on those header pages.

https://github.com/matchboxcowork/matchboxstudio.org/pull/31/commits/ded2cbc961eb24a2723c7e723cbd2d88512f08df is now live on http://104.131.179.41/ as of `Fri Jan 20 10:10:37 EST 2017`.

**Comment by jhewitt12** [comment]
That looks good on the server! I noticed an image file name needed to be changed from .JPG to .jpg on the about.html page so I pulled your commit. It broke everything locally. Do I need to run a rake command or something? Basically it can't find any of the files in assets folder.

**Comment by elliottwilliams** [comment]
`rake clean` followed by `rake` will delete the `build` directory and re-create it, try that.

**Comment by jhewitt12** [comment]
I did that and its still broken. Any other ideas?

**Comment by elliottwilliams** [comment]
How are you serving your dev site? Make sure you're using the server that `guard` starts up (`localhost:9292`). If your server doesn't have `/` as the site's root, then this commit will break asset paths for you. Additionally, verify that you can get an asset from its URL (like `/assets/img/Rendering.png`)

**Comment by jhewitt12** [comment]
When I run it using XAMPP it doesn't work because I had localhost/matchboxstudio.org/. It does work when I serve it without XAMPP. I pushed the img ext change. Other than that I think it all looks good.

**Comment by elliottwilliams** [comment]
Okay, awesome. 
https://github.com/matchboxcowork/matchboxstudio.org/pull/31/commits/026c78925208a4c8a7abff0f856ae84ab1b62e0b is now live on http://104.131.179.41/ as of `Fri Jan 20 11:36:18 EST 2017`.

I'm going to assign @mikelberger as a reviewer to the pull request. Once he looks at the site and approves his review, let's merge and deploy.

**Comment by jhewitt12** [comment]
Sounds great

**Review by mikelberger — COMMENTED** [review]
https://www.dropbox.com/s/9dxlq93xb351bcu/Screenshot%202017-01-20%2014.01.55.png?dl=0

Getting an HTTPS error.

**Comment by elliottwilliams** [comment]
Good catch, I reverted my change that removed `https` prefixed in a couple asset URLs.

**Comment by elliottwilliams** [comment]
https://github.com/matchboxcowork/matchboxstudio.org/pull/31/commits/c5aad4b29dedc6782afab690751553dc0cb5ccd7 live on http://104.131.179.41 as of `Fri Jan 20 14:27:27 EST 2017`.

**Comment by elliottwilliams** [comment]
Here's a task list based on Jason's review of the design changes. I've tried to check off what's already been done, please check off others as you complete them:

- [x] Show pricing information on the application screen
- [x] Remove separate "pricing" and "apply" tabs from main nav
- [x] Add a full-width bar of white space between the street view unit and the background behind it
- [x] Toggle buttons to open/close the street view unit
- [x] Add padding between the map/news row and the contact information, and add padding between contact information and sponsorship rows
- [x] Normalize the colors in sponsor logos and lighten the overall greys
- [x] Changing word-accent color to navy blue
- [x] Title case the byline on the about page
- [x] Change the "+" tab to show testimonies in a single column
  - I need written testominies to proceed here -@elliott

**Comment by elliottwilliams** [comment]
http://104.131.179.41 is live at https://github.com/matchboxcowork/matchboxstudio.org/pull/31/commits/adc121394020dbafca2857160cc80ce2507c48f6 as of `Sun Jan 29 15:03:07 EST 2017`.

**Comment by elliottwilliams** [comment]
I updated the testimonials to what @amandafindlay sent me. You can see the preview at 
http://104.131.179.41/ (showing `4e58e79` as of `Mon Feb 20 22:57:00 EST 2017`).

@mikelberger, can you sign off on this so we can deploy? Thanks!

**Pull request merged**